src/util/xml-helpers.js

```diff
@@ -163,82 +115,73 @@ module.exports = {
         if (!target) return false;
 
         nodes.forEach(node => {
-            const attributes = node.attrib;
-            for (const attribute in attributes) {
-                if (target.attrib[attribute]) {
-                    delete target.attrib[attribute];
-                }
+            for (const attribute in node.attrib) {
+                delete target.attrib[attribute];
             }
         });
 
         return true;
     },
 
     parseElementtreeSync: function (filename) {
-        const contents = stripBom(fs.readFileSync(filename, 'utf-8'));
-        return new et.ElementTree(et.XML(contents));
+        return et.parse(stripBom(fs.readFileSync(filename, 'utf-8')));
     },
 
     resolveParent: function (doc, selector) {
-        let parent, tagName, subSelector;
-
-        // handle absolute selector (which elementtree doesn't like)
-        if (ROOT.test(selector)) {
-            tagName = selector.match(ROOT)[1];
-            // test for wildcard "any-tag" root selector
-            if (tagName === '*' || tagName === doc._root.tag) {
-                parent = doc._root;
-
-                // could be an absolute path, but not selecting the root
-                if (ABSOLUTE.test(selector)) {
-                    subSelector = selector.match(ABSOLUTE)[2];
-                    parent = parent.find(subSelector);
-                }
-            } else {
-                return false;
-            }
-        } else {
-            parent = doc.find(selector);
-        }
-        return parent;
+        if (!selector.startsWith('/')) return doc.find(selector);
+
+        // elementtree does not implement absolute selectors so we build an
+        // extended tree where we can use an equivalent relative selector
+        const überRoot = et.Element('über-root');
+        überRoot.append(doc.getroot());
```

Comment:
`ü` a wrong `u`?

Reply:
No, it's the [right `u`](https://en.wiktionary.org/wiki/uber) 😋. This was just meant as an Easter egg for the review.

We should change it to `metaRoot` for production.

Reply:
```suggestion
        metaRoot.append(doc.getroot());
```